app/styles/_mobile-service-instance-list.less

```diff
@@ -0,0 +1,32 @@
+.mobile-service-instance-list {
+
+  .note {
+    text-align: center;
+  }
+
+  .mobile-service-instance-row {
+    .icon {
+      clear: both;
+      color: #888;
+      font-size: 2em;
+      margin: auto;
+    }
+    .list-pf-name {
+      h3 {
+        .logo {
+          width: 30px;
+          height: 30px;
+          margin: 0 10px;
+          display: block;
+          float: left;
+          font-size: 1.6em;
```

Comment:
`$pf-` var for font-size?

Reply:
Could you point to where the `$pf-` vars are defined please?

Reply:
The console's own vars are in `styles/_variables.less`. The paths to the PF files we are importing can be found in `styles/main.less`.  That will be in `bower_components/patternfly/`.  A good start would be `patternfly/src/less/variables.less`.